### PR TITLE
Fix drops in 1.15

### PR DESCRIPTION
**Pull request by creator3**
## Description
Fix drops in 1.15

## Changes
Replace calls to `block.getDrops()` with calls specifying the correct tool ItemStack

## Related Issues
#1306 #1304 #1303
Some of these issues describe multiple problems with 1.15, not just this one

## Testability
I only did basic testing, but it seems to have fixed the issue. I don't think it'll break any addons, but some of them will probably need the same fix applied

### CODE REVIEW

File: src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/HerculesPickaxe.java [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
 public class HerculesPickaxe extends SimpleSlimefunItem<BlockBreakHandler> {
 
+	// Determines the drops a HerculesPickaxe will get
+	private static final ItemStack EFFECTIVE_PICKAXE = new ItemStack(Material.DIAMOND_PICKAXE);
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/HerculesPickaxe.java]: I think it should be an Iron Pick...

**Comment by WalshyDev** [comment]
God I hate Spigot...

File: src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/ExplosivePickaxe.java [hunk 2]
```diff
@@ -77,7 +80,7 @@ else if (b.getType().name().endsWith("_SHULKER_BOX")) {
 									b.breakNaturally();
 								}
 								else {
-									for (ItemStack drop: b.getDrops()) {
+									for (ItemStack drop: b.getDrops(EFFECTIVE_PICKAXE)) {
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/ExplosivePickaxe.java]: Please add a space before the :
to follow our new conventions.

(Would be awesome if you could change that in all the for loops of the other files you edited too)

**Comment by WalshyDev** [comment]
Are there other getDrops around the plugin that also need to be changed. How do androids work in this regard?

**Comment by creator3** [comment]
The mining talisman also uses getDrops(). I think it's currently broken, though, so maybe a second PR is in order there. I seem to recall I tracked down a bug where the code had been relocated and it ended up that nothing ever happens after it calculates the drops.

CS-CoreLib (v1) also seems to have a call in the deprecated BlockBreaker class. I didn't see anything in CS-CoreLib2, but I was just using a text search for "getDrops()"

**Comment by creator3** [comment]
Oh, also CS-CoreLib2 probably needs a version bump. When I compiled Slimefun, I got a version of CS-CoreLib2 that was incompatible with 1.15 because of Tag.DIRT_LIKE, which was fixed in https://github.com/TheBusyBiscuit/CS-CoreLib2/commit/0f82b794e07ea8ccbe46e03932eca2b56d26dbc9.

**Pull request merged**